clients/admin-ui/src/features/common/table/v2/column-settings/DraggableColumnList.tsx

```diff
@@ -144,7 +144,9 @@ const DraggableColumnListItem = ({
       alignItems="center"
       display="flex"
       minWidth={0}
-      ref={preview}
+      ref={(element) => {
+        preview(element);
+      }}
```

Comment:
Resolving kind of a weird Type issue here.

React's `ref` expects to be passed a React.ReactNode object _or_ to be `undefined`. On the other hand, `preview` is returning a React.ReactNode object _or_ `null`. By explicitly creating a function here, `ref` no longer expects `undefined` and the Types are compatible, since ref passes the element as React.ReactNode _or_ null.

Reply:
Thanks for the explanation!